forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -1211,6 +1211,33 @@ public static void addTriggerAbility(final KeywordInterface inst, final Card car
             trigger.setOverridingAbility(AbilityFactory.getAbility(effect, card));
 
             inst.addTrigger(trigger);
+            
+        } else if (keyword.equals("For Mirrodin")) {
+            final StringBuilder sbTrig = new StringBuilder();
+            sbTrig.append("Mode$ ChangesZone | Destination$ Battlefield | ");
+            sbTrig.append("ValidCard$ Card.Self | Secondary$ True | TriggerDescription$ ");
+            sbTrig.append("For Mirrodin (").append(inst.getReminderText()).append(")");
+
+            final StringBuilder sbRebel = new StringBuilder();
+            sbRebel.append("DB$ Token | TokenAmount$ 1 | TokenScript$ r_2_2_rebel |TokenOwner$ You | RememberTokens$ True");
```

Comment:
`| TokenOwner` (spacing)

Reply:
TokenAmount as is won't do any damage, but default is 1 so you can safely remove it here to shorten the line a bit